releases/v1.3.0-beta.toml

```diff
@@ -20,31 +20,62 @@ and pluggable. On Windows, this release brings a new runtime utilizing the shim
 API. For clients, there are many new features and improvements completely
 implemented in the client libraries without requiring daemon upgrade.
 
-## Runtime
-* New Windows runtime using hcsshim
-* Improvements to ttrpc for daemon to shim communication (https://github.com/containerd/containerd/pull/3341)
-
-## Snapshots
-* Devmapper snapshotter (https://github.com/containerd/containerd/pull/3022)
-* Improved label support for plugins
-
-## Plugins
-* Support for plugins registering as a TCP service
-* Configurable plugin directory
-
-## CRI
-
-## Client
-* Default handling from namespace labels (https://github.com/containerd/containerd/pull/3403)
-* Improved Docker resolver with mirroring support
-* Support for cross repository push (https://github.com/containerd/containerd/pull/3218)
-
-## Other
-* [API] Add support for direct resource management in leases (https://github.com/containerd/containerd/pull/3304)
-* [API] Add ttrpc service for shim event publishing
-* [API] Add annotations to descriptors in API
-* [API] Add id to TaskDelete event message to match exec id
-* Support additional garbage collection labels
+### Runtime
+* **New Windows V2 runtime using [hcsshim](https://github.com/Microsoft/hcsshim)**
```

Comment:
@dmcgowan - Can we make this `[hcsshim](https://github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1)`

Reply:
> https://github.com/Microsoft/hcsshim/cmd/containerd-shim-runhcs-v1

It doesn't work. Probably you mean https://github.com/microsoft/hcsshim/tree/master/cmd/containerd-shim-runhcs-v1

Reply:
Haha yep. Thats what I get for not checking the link I type :)

Reply:
Sure, but can you add a README file under the `cmd` there which explains how to use it with containerd then?

Reply:
Yes I will